src/firefly/java/edu/caltech/ipac/firefly/data/ServerParams.java

```diff
@@ -58,6 +58,7 @@
     public static final String ID = "id";
     public static final String BID = "bid";
     public static final String CHANNEL_ID = "channelID";
+    public static final String TRY_MS = "tryMS";
```

Comment:
My guess is MS stands for milliseconds, but I had to look how it's used. We might want to document the meaning of some parameters.